x-pack/plugins/apm/server/lib/apm_telemetry/index.ts

```diff
@@ -26,34 +25,28 @@ export function createApmTelementry(
 }
 
 export async function storeApmServicesTelemetry(
-  server: APMLegacyServer,
+  savedObjectsClient: InternalSavedObjectsClient,
```

Comment:
uses the savedObjectsClient directly rather than obtaining a new instance from the legacy kibana server